devops/cloud-functions/gcf-pin-ipfs-hash/ipfs-cluster-api-service.js

```diff
@@ -1,15 +1,18 @@
 const fetch = require('cross-fetch')
 
+// Per Protocol Labs request, we add this param to any request we send to their cluster.
+const ORIGIN_PARAM = '?name=OriginProtocol'
+
 class IpfsClusterApiService {
   constructor(ifpsClusterUrl, username, password) {
     this.ifpsClusterUrl = ifpsClusterUrl
```

Comment:
We might as well fix this typo in this PR too?

Reply:
Yep. might as well ! done